tests/schemas/test_compat.py

```diff
@@ -5,24 +5,13 @@
 import json
 
 import tests.schemas
-from tests.schemas.builder import (
-    generate_api_data_specs,
-    generate_api_protocol_specs,
-    generate_core_data_specs,
-)
+from tests.schemas.builder import generate_api_data_specs, generate_core_data_specs
 
 
 # /!\ Those tests only deal with pure Python schemas (even if running with --runrust, /!\
 # /!\ even if the Rust schemas are the one exposed by core and api packages)          /!\
 
 
-def test_api_protocol_compat():
```

Comment:
This is removed becaue protocol is now 100% in Rust ?

Reply:
yes